corehq/messaging/scheduling/forms.py

```diff
@@ -406,6 +417,8 @@ def add_initial_recipients(self, recipients, initial):
             'user_organization_recipients': ','.join(user_organization_recipients),
             'case_group_recipients': ','.join(case_group_recipients),
             'include_descendant_locations': self.initial_schedule.include_descendant_locations,
+            'restrict_location_types': 'Y' if len(self.initial_schedule.location_type_filter) > 0 else 'N',
+            'location_types': ','.join([str(i) for i in self.initial_schedule.location_type_filter]),
```

Comment:
This should use unicode (`six.text_type`) rather than `str`.

Nit: square brackets (list comprehension) are not necessary/could be removed to create a generator expression.

Reply:
updated: https://github.com/dimagi/commcare-hq/pull/20107/commits/7109e01d489e1d99e8bc3b02a94377a5c4af7c5a